### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by amashutikov**
- [DEMO LINK](https://amashutikov.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,39 @@
...
+      <div class="calendar__day calendar__day--22"></div>
+      <div class="calendar__day calendar__day--23"></div>
+      <div class="calendar__day calendar__day--24"></div>
+      <div class="calendar__day calendar__day--25"></div>
+      <div class="calendar__day calendar__day--26"></div>
+      <div class="calendar__day calendar__day--27"></div>
+      <div class="calendar__day calendar__day--28"></div>
+      <div class="calendar__day calendar__day--29"></div>
+      <div class="calendar__day calendar__day--30"></div>
+      <div class="calendar__day calendar__day--31"></div>
```
Comment (okosohor) [inline src/index.html]: You don`n need to add number for day you can use nth-child.

**Review by okosohor — APPROVED** [review]

**Review by silchyn — CHANGES_REQUESTED** [review]
fix comment above

**Review by vitvadym — APPROVED** [review]
nice👍 

first point task: Display a calendar in the middle of the screen)
check it 👀

File: src/index.html [hunk 2]
```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day-fri calendar--month-length-29 page__calendar"
```
Comment (silchyn) [inline src/index.html:10]:
```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length--29 page__calendar"
```

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
+@import "./utils/variables";
+@import "./utils/mixins";
```
Comment (silchyn) [inline src/styles/blocks/calendar.scss:2]: shouldn't be there

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    @include squre($cell-size);
+
+    display: flex;
+    justify-content: center;
+    line-height: 100px;
+
+    transition: 0.5s;
+  }
+
+  &__day:hover {
```
Comment (silchyn) [inline src/styles/blocks/calendar.scss:47]:
```suggestion
    &:hover {
```

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**